Libraries/Experimental/SwipeableRow/SwipeableQuickActions.js

```diff
@@ -10,11 +10,16 @@
 
 'use strict';
 
-const DeprecatedViewPropTypes = require('DeprecatedViewPropTypes');
 const React = require('React');
 const StyleSheet = require('StyleSheet');
 const View = require('View');
 
+import type {ViewStyleProp} from 'StyleSheet';
+
+export type SwipeableQuickActionsProps = $ReadOnly<{|
```

Comment:
Does this need to be exported? We try not to export a bunch of things, only the things that are actually necessary or expected to be used by other files.

Reply:
That makes sense! Reducing the public api surface area is a good thing.